stripe-samples/accept-a-payment
# Add an example with Link

This PR adds an example to showcase Link in the `custom-payment-flow` sample.

I updated the servers in all 7 programming languages but could do with some help with the PHP one. When the payment using Link is successful, it should redirect to `/success` and I couldn't figure out how to implement that.

Steps to review this PR:
1. Fetch this branch
2. Run one of the servers
3. Select the Link example
- [ ] The email field should be displayed
4. If you **don't** already have an account with Link, enter your email address.
   - [ ] A field should appear at the bottom of the form to save your information to pay faster with Link
   - Enter test credit card details and click `Pay`
   - Restart the process and enter your email
   - [ ] The field to submit an authentication code should appear
   - Enter `424242`
   - [ ] Your email should be recognized and the card details should be automatically filled
   - [ ] Click `Pay` and the payment should be successful

If you **do** have an account with Link, enter your email address. 
   - [ ] The field to submit an authentication code should appear
   - Enter `424242`
   - [ ] Your email should be recognized and the card details should be automatically filled
   - [ ] Click `Pay` and the payment should be successful

5. Repeat with each of the other servers
---

*Note: I only updated the `html` client for now to avoid making this PR too large.*
